### PR TITLE
Remove DNX branding from footer

### PR DESCRIPTION
**Pull request by JunTaoLuo**
#722 I also did a quick search for other occurrences of the word dnx. Still need to investigate the use of https://github.com/aspnet/Hosting/blob/12fff1833fdcaa36b6d43bbdeb0465a11d9cf365/src/Microsoft.AspNetCore.Server.Testing/Deployers/IISExpressDeployer.cs#L88-L95.

The new error footer looks like:
image

Review: @Tratcher 
Approval: @Eilon @DamianEdwards

### CODE REVIEW

**Comment by davidfowl** [comment]
The version is wrong but it's the "rightest" thing we can do right now 😄 

:shipit: !

**Comment by DamianEdwards** [comment]
Shouldn't be ".NET Framework CoreCLR", that's combining two things that shouldn't be. It's either ".NET Framework version 4.*" or ".NET Core version *.*".

Also, the "x64 Runtime version 4.0.0.0" is confusing. What is that referring to exactly? We don't have DNX anymore so I don't think it's even needed. Move the bitness string into the first string and remove that section completely.

For the OS string, please include the bitness there too, as it may be different to the runtime bitness.

**Comment by JunTaoLuo** [comment]
@DamianEdwards Just want to clarify a few things here. 

1. ".NET Framework" and ".NET Core" makes sense but what about Mono? Since we differentiate it in `PlatformAbstractions`, we originally had ".NET Framework Mono" but maybe we can drop the "Mono" part of it. 
2. The runtime version is what we get from `PlatformAbstractions`. There it uses the version of the assembly containing the type `object` as the "Runtime" version. I'm guessing this is what was meant by it's wrong but it's the rightest thing for now.
3. I don't think hosting is the right place to figure out the OS bitness. `PlatformAbstractions` feels like the more logical place to have this logic. There is a work item to update to using CoreFX's `RuntimeInformation` which will have this information but that's for RTM. Unless we are going to pull that back to rc2, I'll look into some other options.

**Comment by DamianEdwards** [comment]
Let's just fix the first one for RC2 then. Happy to keep "Mono" where applicable, so ".NET Framework", ".NET Core", and ".NET Framework (Mono)"?

**Comment by JunTaoLuo** [comment]
@DamianEdwards I also removed the runtime section as well and moved the bitness string to the clr type section. The updated footer looks like 
image
image
 
Do you like this better?

**Comment by muratg** [comment]
I think that's fine, only part that's a little weird is ".NET Framework x64 version 4.0....".  @DamianEdwards thoughts?

File: src/Microsoft.AspNetCore.Hosting/Startup/StartupExceptionPage.cs [hunk 1]
```diff
@@ -234,6 +234,7 @@ private static string GenerateFooterEncoded()
         {
             var environment = PlatformServices.Default.Runtime;
             var runtimeType = HtmlEncodeAndReplaceLineBreaks(environment.RuntimeType);
+            var runtimeDisplayName = runtimeType == "CoreCLR" ? ".NET Core" : runtimeType == "CLR" ? ".NET Framework" : ".NET Framework Mono";
```
Comment (davidfowl) [inline src/Microsoft.AspNetCore.Hosting/Startup/StartupExceptionPage.cs]: Change .NET Framework Mono -> Mono?

**Comment by JunTaoLuo** [comment]
:up::date: as per offline discussion with @DamianEdwards 

The updated footer looks like:
new clr exception
new coreclr exception
 Will create a followup issue to convert the full CLR version to something more reasonable like 4.5.1, 4.6...

**Pull request merged**